en/docs/references/extend/localization-support-in-identity-server.md

```diff
@@ -67,3 +67,12 @@ applying the same above steps 2 through 6 in the previous section
 starting with the following location.
 
 `<IS_HOME>/repository/deployment/server/webapps/accountrecoveryendpoint/WEB-INF/classes/org/wso2/carbon/identity/mgt/recovery/endpoint/i18n/`
+
+## Configuring Localization for My Account and Console
+
+We maintain the i18n (localization) resources for My Account and Console in a separate module inside the
+[identity-apps](https://github.com/wso2/identity-apps/tree/v1.0.558) repository hence the localization configurations
```

Comment:
why are we referring to this particular version of the repo? 
`v1.0.558`?

@deshankoswatte

Reply:
That is the version which is related to the WSO2 Identity Server 5.11.0 release. However, this should be changed later when the WSO2 Identity Server 6.0.0 is released.

Reply:
@piraveena Is this the correct version number for IS 6.0.0?

Reply:
@nilminiwso2 This is not the correct number for IS 6.0.0. The number should be taken after the IS 6.0.0 is released.